search/src/Components/auth/auth_error.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+const AuthError = (props) => {
+    const { error } = props;
+    return (
+        <div>
+            { error ? <p style={{color: 'red', margin: 0}}>{ error }</p> : ""}
```

Comment:
React training might also help us do a better job of theming, rather than having local custom styles.